convert_hf_to_gguf.py

```diff
@@ -5284,13 +5284,14 @@ def set_vocab(self):
         self.gguf_writer.add_token_type_count(self.hparams.get("type_vocab_size", 1))
 
         # convert to phantom space vocab
-        def phantom(tok):
-            if tok.startswith("[") and tok.endswith("]"):
+        def phantom(tok, toktype):
+            if toktype != gguf.TokenType.NORMAL:
```

Comment:
I'm not entirely sure if the condition `toktype != gguf.TokenType.NORMAL` will correctly identify all special tokens.